app.json

```diff
@@ -18,5 +18,10 @@
         "IPostLivechatAgentUnassigned",
         "IPostLivechatRoomClosed",
         "IUIKitLivechatInteractionHandler"
+    ],
+    "permissions": [
+        {
+            "name": "scheduler"
+        }
```

Comment:
This will enable the permission system and as we haven't specified all the permissions(msg, room read/write etc) we are using in the app. It will break the bot. 

We can remove it for now. https://developer.rocket.chat/apps-development/permission-system#notice-about-rule-enforcement